firebase-crashlytics/src/androidTest/java/com/google/firebase/crashlytics/core/CrashlyticsCoreTest.java

```diff
@@ -728,7 +728,7 @@ public boolean accept(File file) {
     }
   }
 
-  private static class AnySessionIdDirectoryFilter extends DirectoriesOnlyFilter {
+  private static class AnySessionIdDirectoryFilter extends SessionDirectoriesOnlyFilter {
     private static String ANY_SESSION_ID_REGEX =
```

Comment:
just an fyi, with regex, I like to give an example of typical cases in javadocs nearby. Any chance we could add one here?

Reply:
Test class, plus this code is slated for removal down the track.